docs/general/endpoints.rst

```diff
@@ -156,42 +151,29 @@ models.
 
 By this restriction, additional _features_ are calculated and returned:
 
-- ``error-transformed``:
-    - Error per feature/tag calculated from the transformed values
-- ``error-untransformed``:
-    - Error per feature/tag calculated from the ``inverse-transformed-model-output`` against ``original-input``
-- ``inverse-transformed-model-output``:
-    - The ``model-output`` which is ran backward through pipeline, which "untransforms" it to the same level
-      as the ``original-input``; this assumes that all steps in the pipeline support ``.inverse_transform``
-- ``total-transformed-error``:
-    - This is the vector norm of the error between a sample output and the sample input, using the transformed values
-- ``total-untransformed-error``:
-    - This is the vector norm of the error between a sample output and the sample input, using the inverse-transformed values
+- ``tag-anomaly``:
+    - Anomaly per feature/tag calculated from the expected tag input (y) and the model's output for those tags (yhat)
+- ``total-anomaly``:
+    - This is the total anomaly for the given point as calculated by the model.
 
 Sample response:
 
 .. code-block:: python
 
     {'data': [{'end': [None],
-               'error-transformed': [2.746687859183499,
-                                     2.4497416272485886,
-                                     2.508896707372706],
-               'error-untransformed': [36.94235610961914,
-                                       33.25593566894531,
-                                       34.43174743652344],
-               'inverse-transformed-model-output': [37.94235610961914,
-                                                    35.25593566894531,
-                                                    37.43174743652344],
-               'model-output': [0.6726480722427368,
-                                0.5305765271186829,
-                                0.7146536707878113],
+               'tag-anomaly': [
+                    2.746687859183499,
+                    2.4497416272485886,
+                    2.508896707372706
+               ],
+               'total-anomaly': [60.4668517758241],
+               'model-output': [
+                    0.6726480722427368,
```

Comment:
These are not actually the model outputs now right? Since we are now training the models to output X, and not X-scaled, these should be in the original high scales?

Reply:
ya, I guess I'm not really paying attention too much to the _values_ but the _keys_ here.